Examples/2048/Game2048.js

```diff
@@ -29,9 +32,15 @@ var Animated = require('Animated');
 var GameBoard = require('GameBoard');
 var TouchableBounce = require('TouchableBounce');
 
-var BOARD_PADDING = 3;
-var CELL_MARGIN = 4;
-var CELL_SIZE = 60;
+var BOARD_PADDING = Platform.isTVOS ? 9 : 3;
+var CELL_MARGIN = Platform.isTVOS ? 12 : 4;
+var CELL_SIZE = Platform.isTVOS ? 180 : 60;
+var BORDER_RADIUS = Platform.isTVOS ? 15 : 5;
+
+var SIZE_40 = Platform.isTVOS ? 120 : 40;
+var SIZE_24 = Platform.isTVOS ? 72 : 24;
+var SIZE_20 = Platform.isTVOS ? 60 : 20;
+var SIZE_18 = Platform.isTVOS ? 54 : 18;
```

Comment:
Might be easier to inline all of these conditions directly into the styles below? That way there's no need to add the extra lines.